### DISCUSSION TITLE
Invalid communication disabled timestamp (timeout member)

### DISCUSSION THREAD

**Discussion by Matieus** [Pycord-Development/pycord#787] [Q&A]
When you try to send a timeout request when "until - discord.utils.utcnow()" is longer than 28 days, it returns this error.
    **discord.errors.HTTPException: 400 Bad Request (error code: 50035): Invalid Form Body
    In communication_disabled_until: Invalid communication disabled timestamp**

I know this is limited by discord, but can you create a separate error for that?

**Comment by plun1331** [comment]
I don't see a point, you should be responsible for passing the correct parameters.
This discussion would also be better off as an issue.

**Comment by Lulalaby** [comment]
No we can't. As @plun1331 said.
We give you the exact error which discord returns.
Otherwise we had to handle every new change and discord loves to change things.
